src/lib/y2storage/autoinst_profile/skip_list_value.rb

```diff
@@ -68,11 +107,73 @@ def name
         disk.basename
       end
 
+      # @return [String,nil] DASD format or nil if not a DASD device
+      #   backward compatibility
+      def dasd_format
+        return nil unless disk.is_a?(Y2Storage::Dasd)
```

Comment:
It's better to use `unless disk.is?(:dasd)`. It's more duck-typing oriented and doesn't depend on the concrete hierarchy enforced by libstorage-ng at the moment.